exercises/diamond/test/test_diamond.c

```diff
@@ -0,0 +1,155 @@
+#include "vendor/unity.h"
+#include "../src/diamond.h"
+#include <stdlib.h>
+
+#define ARRAY_SIZE(arr) (sizeof(arr)/sizeof(arr[0]))
+
+void setUp(void)
+{
+}
+
+void tearDown(void)
+{
+}
+
+static void free_all(char **diamond)
+{
+   free(diamond[0]);
+   free(diamond);
+}
+
+void test__degenerate_case_with_a_single_a_row(void)
```

Comment:
I noticed that in this test file all tests are prefixed with `test__` instead of `test_`. Can you update this?

Reply:
Indeed. Sorry about this.